modules/builtin/src/content-types/file.js

```diff
@@ -0,0 +1,83 @@
+const base = require('./_base')
+const path = require('path')
+const utils = require('./_utils')
+
+function render(data) {
+  const events = []
+
+  if (data.typing) {
+    events.push({
+      type: 'typing',
+      value: data.typing
+    })
+  }
+
+  return [
+    ...events,
+    {
+      type: 'file',
+      title: data.title,
+      url: utils.formatURL(data.BOT_URL, data.file),
+      collectFeedback: data.collectFeedback
+    }
+  ]
+}
+
+function renderElement(data, channel) {
+  // These channels now use channel renderers
+  if (['telegram', 'twilio', 'slack', 'vonage'].includes(channel)) {
+    return utils.extractPayload('file', data)
+  }
+
+  return render(data)
+}
+
+module.exports = {
+  id: 'builtin_file',
+  group: 'Built-in Messages',
+  title: 'module.builtin.types.file.title',
+
+  jsonSchema: {
+    description: 'module.builtin.types.file.description',
+    type: 'object',
+    $subtype: 'file',
+    required: ['file'],
+    properties: {
+      file: {
+        type: 'string',
+        $subtype: 'file',
+        $filter: '.pdf',
```

Comment:
Why is there a pdf filter here? Shouldn't we allow any file types for a file content type?

Reply:
It's the current use case, but I agree with you, that we should probably allow any type of file in the future!